index.js

```diff
@@ -0,0 +1,6 @@
+import BookManager from './modules/bookManager.js';
+// eslint-disable-next-line no-unused-vars
+import displayDate from './modules/displayDate.js';
+
+// eslint-disable-next-line no-unused-vars
+const bookManager = new BookManager();
```

Comment:
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but please note that it's not recommended to disable linter checks manually using inline comments like the one above.